docs/UserGuide.md

```diff
@@ -151,82 +153,170 @@ LibTask's data is saved in the hard disk automatically after any command that ch
-* The order of the keywords does not matter. e.g. results from the keyword `Hans Bo` will match results of the keyword `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. results from the keyword `Han` will not match results from the keyword `Hans`
-* Patrons matching at least one part of the keyword will be returned (i.e. `OR` search).
-  e.g. keyword `Hans Bo` will return patrons with names `Hans Gruber`, `Bo Yang`
+![patron-edit-1](images/patron-edit-1.png)
+
+After entering the command, the first patron has its name edited to `John Cena`, phone number edited to `91959491` and email edited to `johncena@u.nus.edu`
+
+![patron-edit-2](images/patron-edit-2.png)
+
+To remove all tags and edit the first patron's name from `Alex Yeoh` to `John Cena`, its phone number from `87438807` to `91959491` and email from `alexyeoh@example.com` to `johncena@u.nus.edu`you can enter the following command:
 
-Example:
+`patron edit 1 n/John Cena p/91959491 e/johncena@u.nus.edu t/`
 
-`patron find n/alex n/david` returns `Alex Yeoh`, `David Li`<br>
+Before entering the command, the first patron has different tags.
 
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+![patron-edit-3](images/patron-edit-3.png)
+
+After entering the command, the first patron will have their tags removed.
+
+![patron-edit-4](images/patron-edit-4.png)
 
 #### 3.2.5. Deleting a patron : `patron delete`
 
-Deletes the patron at a specified index in the displayed patron list.
+To delete a specific patron, you can enter the `delete` command with the format shown below.
 
 Format: `patron delete INDEX`
 
+**Notes about the delete command:**
 * Deletes the patron at the specified `INDEX`.
 * The index refers to the index number shown in the displayed patron list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
-Examples:
-* `patron list` followed by `patron delete 2` deletes the 2nd patron shown in the displayed patron list.
-* `patron find n/Betsy` followed by `patron delete 1` deletes the 1st person in the results of the `find` command.
+**Example:**
+
+To delete the second patron, you can enter the following command:
+
+`patron delete 2`
+
+Before entering the command, the second patron in LibTask is `Bernice Yu`.
+
+![patron-delete-1](images/patron-delete-1.png)
+
+After entering the command, patron `Bernice Yu` is deleted.
+
+![patron-delete-2](images/patron-delete-2.png)
 
 #### 3.2.6. Listing all patrons with overdue books : `patron overdue`
 
-Lists all patrons in LibTask's patron list with overdue books.
+To list all patrons with overdue books, you can enter the `overdue` command with the format shown below.
 
 Format: `patron overdue`
 
+**Example:**
+
+Before entering the command, all patrons are displayed.
+
+![patron-list-1](images/patron-list-1.png)
+
+After entering the command, only patrons with overdue books are displayed.
+
+![patron-list-2](images/patron-list-2.png)
+
```

Comment:
This line is logical in explaining to the readers of what to expect from running the patron overdue command :100: